tests/unit/common/test_generators.py

```diff
@@ -45,4 +45,23 @@ def test_multivariate_vector_generation_2d(self, mean, cov):
 
 
 class TestConstantGenerator:
-    pass
+    @pytest.mark.parametrize(
+        "constant",
+        [
+            (np.array([1]),),
+            (np.array([1, 2]),),
+            (np.array([1, 2, 3]),),
+            (np.array([[1, 1], [1, 1]]),),
+            (np.array([[1, 2, 3], [3, 2, 1]]),),
+        ],
+    )
```

Comment:
It would be a good idea to add tests for using scalar values in addition to numpy arrays.